src/App.css

```diff
@@ -1,38 +1,272 @@
+  95%   {color: var(--color-5)}
+  100%  {color: var(--color-1)}
+}
+
+header p {
+  font-size: clamp(0.8rem, 3vw, 1.1rem);
+  line-height: 1.5rem;
+}
+
+form {
   display: flex;
+  flex-wrap: wrap;
   flex-direction: column;
+  justify-content: center;
   align-items: center;
+  text-transform: uppercase;
+}
+
+.options {
+  display: flex;
+  flex-wrap: wrap;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  margin: 20px 0;
+}
+
+.option {
+  display: flex;
+  align-items: center;
+  margin: 10px 20px;
+}
+
+label {
+  font-size: 0.8rem;
+  text-transform: capitalize;
+}
+
+input[type="color"] {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  box-shadow: .2em .2em .2em rgba(0, 0, 0, .1);
+  border: 0;
+  background: none;
+  cursor: pointer;
+  padding: 0;
+  width: 100px;
+  height: 30px;
+  margin: 0 20px;
+}
+
+input[type="text"]:focus{
+  outline: none;
+}
+
+button {
+  font-family: 'Comfortaa', cursive;
+  background: white;
+  border: none;
+  padding: 10px;
+  cursor: pointer;
+  border-radius: 5px;
+}
+
+.submitButton {
+  text-transform: uppercase;
+  letter-spacing: 5px;
+  width: 80%;
+  margin: 20px 0;
+  box-shadow: .5em .5em .5em rgba(0, 0, 0, .1);
+  transition: 0.5s ease;
+  font-size: clamp(0.8rem, 2vw, 1.1rem);
+}
+
+.submitButton:hover {
+  background-color: black;
+  color: white;
+  letter-spacing: 8px;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  border: 1px solid transparent;
+  box-shadow: 1em 1em 1em rgba(0, 0, 0, .1);
+  padding-left: 0;
+  background-color: white;
+  margin: 20px 0;
+  position: relative;
+  border-radius: 5px;
+}
+
+.removeButton {
+  background-color: black;
   color: white;
+  text-align: center;
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  top: 5px;
+  right: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  font-size: 1.2rem;
+  transition: all 0.5s ease;
 }
 
-.App-link {
-  color: #61dafb;
+.removeButton:hover, 
+.removeButton:focus {
+  width: 33px;
+  height: 33px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+li {
+  padding: 20px;
+  font-size: 0.8rem;
+}
+
+.results {
+  width: 100px;
+  height: 100px;
+  border: 1px solid lightgray;
+  border-radius: 50%;
+}
+
+footer {
+  margin-top: 80px;
+  font-size: 0.8rem;
+}
+
+a {
+  text-decoration: none;
+}
+
+
+@media (max-width: 800px){
+
+  input[type="color"] {
+    margin: 0 10px;
   }
-  to {
-    transform: rotate(360deg);
+
+  header p {
+    line-height: 1.1rem;
+  }
+
+  .submitButton {
+    margin: 10px 0;
+  }
+}
+
+
+@media (max-width: 550px){
+
+  .options {
+    margin: 0;
+  }
+
+  .submitButton {
+    width: 100%;
+  }
+
+  .results {
+    width: 70px;
+    height: 70px;
+  }
+
+  ul {
+    padding-top: 30px;
+  }
+
+  ul p {
+    margin: 5px 0;
+  }
+
+  li {
+    padding: 5px;
+    font-size: 0.7rem;
+  }
+  
+  .submitButton:hover {
```

Comment:
I was looking at the reponsive version with around 400px width and the button letter spacing feels like a lot. it may be a me thing but if i look at it for the first time, the spacing between the words and letters kinda mixes up for me!